CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+### 0.6.0 (unreleased)
+
+- New features
+  - New  `ProductHint` trait allowing customization of the derived `ConfigConvert` for case classes, superseeding
+    `ConfigFieldMapping` ([docs](https://github.com/melrief/pureconfig#override-behaviour-for-case-classes)). In
+    addition to defining field name mappings, `ProductHint` instances control:
+    - Whether default field values should be used when
+      fields are missing in the config ([docs](https://github.com/melrief/pureconfig#default-field-values));
+    - Whether unknown keys are ignored or cause pureconfig to return a `Failure`
+      ([docs](https://github.com/melrief/pureconfig#unknown-keys)).
+- Breaking changes
+  - `ConfigFieldMapping` has no type parameters now;
+  - `ConfigFieldMapping` was replaced by `ProductHint` as the type of object to put in scope in order to customize
+    the derivation of `ConfigConvert` for case class. Old `ConfigFieldMapping` implicit instances in scope have no
+    effect now. The migration can be done by replacing code like
+    `implicit def mapping: ConfigFieldMapping[T] = <mapping>` with
+    `implicit def productHint: ProductHint[T] = ProductHint(<mapping>)`;
+  - The default field mapping changed from camel case config keys (e.g. `exampleKey`) to kebab case keys (e.g.
```

Comment:
This probably ought to be the top item in the Breaking list for this release. It is more likely to affect users than the one above.
Pushing it down feels like feels like burying the lede.